SecurityService.OpenIdConnect.IntegrationTests/Common/DockerHelper.cs

```diff
@@ -5,104 +5,230 @@
+                                                                            .UseImage("securityservicetestwebclient").ExposePort(5004)
+                                                                            .UseNetwork(new List<INetworkService>
+                                                                                        {
+                                                                                            networkService
+                                                                                        }.ToArray())
+                                                                            .Build().Start();//.WaitForPort("5004/tcp", 30000);
+
+            return securityServiceTestUIContainer;
+
+            //Console.Out.WriteLine("Started Security Service");
+        }
+
+        private readonly NlogLogger Logger;
+        protected List<INetworkService> TestNetworks;
+        protected List<IContainerService> Containers;
         public Guid TestId;
-        private void SetupTestNetwork()
+        public String SecurityServiceContainerName;
+        public String SecurityServiceTestUIContainerName;
+        protected Int32 SecurityServicePort;
+        public Int32 SecurityServiceTestUIPort;
+        public ISecurityServiceClient SecurityServiceClient;
+
+        public DockerHelper(NlogLogger logger)
         {
-            // Build a network
-            this.TestNetwork = new Ductus.FluentDocker.Builders.Builder().UseNetwork($"testnetwork{this.TestId}").Build();
+            this.Logger = logger;
+            this.Containers = new List<IContainerService>();
+            this.TestNetworks = new List<INetworkService>();
         }
 
-        public async Task StartContainersForScenarioRun(String scenarioName)
+        public override async Task StartContainersForScenarioRun(String scenarioName)
         {
-            String traceFolder = $"/home/txnproc/trace/{scenarioName}/";
+            String traceFolder = FdOs.IsWindows()
+                ? $"C:\\home\\txnproc\\trace\\{scenarioName}"
+                : $"/home/txnproc/trace/{scenarioName}";
 
             Logging.Enabled();
 
             Guid testGuid = Guid.NewGuid();
             this.TestId = testGuid;
 
+            (String, String, String) dockerCredentials = ("https://www.docker.com", "stuartferguson", "Sc0tland");
+
             // Setup the container names
+            this.SecurityServiceContainerName = $"securityservice{testGuid:N}";
             this.SecurityServiceTestUIContainerName = $"securityservicetestui{testGuid:N}";
 
-            this.SetupTestNetwork();
-            
-            this.SetupSecurityServiceTestUIContainer(traceFolder);
-            this.SecurityServiceTestUIPort = this.SecurityServiceTestUIContainer.ToHostExposedEndpoint("5004/tcp").Port;
+            //INetworkService testNetwork = DockerHelper.SetupTestNetwork();
```

Comment:
Codacy found an issue: [Remove this commented out code.](https://app.codacy.com/manual/stuart_ferguson1/SecurityService/pullRequest?prid=6144581)